### ISSUE TITLE
Add tooltips to install buttons

### ISSUE THREAD

**Issue by pwalm** [mozilla/addons-frontend#1046]
Relates to #1153

We need tooltips on disco pane! @designakt I think you were working on this with Michelle?

**Comment by designakt** [comment]
Yes. @atsay and @devaneymoz have been writing them.
Michelle only looked at for Firefox Voice and Tone.

Those are the strings that we should use:

Toggle Install Theme:
`Install this theme on Firefox.`
Click Theme Image:
`Install this theme on Firefox.`
Toggle Uninstall Theme:
`Uninstall this theme.`
Click Theme Name & Author:
`See more details about this theme.`

Toggle Install Extension:
`Install this extension on Firefox.`
Toggle Uninstall Extension:
`Uninstall this extension.`
Click Extension Name & Author:
`See more details about this extension.`

Click AMO:
none
Click Video:
none

https://docs.google.com/document/d/1rGBaMwpr_qbdah_LE1oZkTVIPZKNEAwRfH0zRbd0YaY/edit#heading=h.cjv7fq6632sd

**Comment by atsay** [comment]
Thanks for adding the strings Markus! @pwalm any other info you need for this?

**Comment by pwalm** [comment]
Nope! Just wanted to file the issues so we all had a touch point 👍🏻

**Comment by aayushsanghavi** [comment]
@pwalm I could add these tool tips. Could you tell me what file I should be looking at?

**Comment by tofumatt** [comment]
@aayushsanghavi I don't think the tooltip design is ready yet.

**Comment by aayushsanghavi** [comment]
Okay

**Comment by pwalm** [comment]
@tofumatt I don't think this needs any design, aren't we just going with platform-styled tool tips?

**Comment by tofumatt** [comment]
The issue with that is they won't work on mobile because platform tool tips (eg the title attribute) appears on hover. Which mobile doesn't have.

But there are basic white on black libraries that do this already. We could probably do one, screenshot it, and you could r? it and give feedback. Let's try that and see.

Matthew Riley MacPherson (sent from mobile)

**Comment by pwalm** [comment]
Awesome, let's try it. Thanks @tofumatt
